beacon-chain/core/blocks/block_operations.go

```diff
@@ -891,7 +924,25 @@ func ProcessTransfers(
 	verifySignatures bool,
 ) (*pb.BeaconState, error) {
 	transfers := body.Transfers
-
+	if uint64(len(transfers)) > params.BeaconConfig().MaxTransfers {
+		return nil, fmt.Errorf(
+			"number of transfers (%d) exceeds allowed threshold of %d",
+			len(transfers),
+			params.BeaconConfig().MaxTransfers,
+		)
+	}
+	// Verify there are no duplicate transfers.
```

Comment:
we don't need this anymore, it's checked in `process_operation`:
https://github.com/prysmaticlabs/prysm/blob/spec-v0.6/beacon-chain/core/state/transition.go#L267

Reply:
Can we address these later? I was just trying to fix these types and I had test failures due to newly missing validation.

Reply:
sure, I was planning to make a PR that takes all the takes to `process_operation` level

Reply:
I've made a PR to handle this with tests as well #2900 👍